app/components/temporal-selector.hbs

```diff
@@ -0,0 +1 @@
+<AuToggleSwitch @label={{t "trafficMeasureConcept.attr.temporal"}} @disabled={{false}} @checked={{@concept.valid}} @onChange={{this.update}}/>
```

Comment:
this toggles the wrong property, it's toggling `@concept.valid` instead of the temporal property...